Services/HetrixtoolsService/AzineMailgunHetrixtoolsService.php

```diff
@@ -0,0 +1,98 @@
+<?php
+
+namespace Azine\MailgunWebhooksBundle\Services\HetrixtoolsService;
+
+/**
+ * AzineMailgunHetrixtoolsService
+ *
+ * This service is a wrapper for using Hetrixtools blacklist check functionality https://hetrixtools.com/.
+ */
+class AzineMailgunHetrixtoolsService
+{
+    /**
+     * @var string
+     */
+    private $apiKey;
+
+    /**
+     * @var string
+     */
+    private $blacklistIpCheckUrl;
+
+    /**
+     * AzineMailgunHetrixtoolsService constructor.
+     *
+     * @param string $apiKey
+     * @param string $url
+     */
+    public function __construct($apiKey, $url)
+    {
+        $this->apiKey = $apiKey;
+        $this->blacklistIpCheckUrl = $url;
+    }
+
+    /**
+     * @param string $ip
+     * @return HetrixtoolsServiceResponse $response
+     */
+    public function checkIpAddressInBlacklist($ip)
+    {
+        $url = $this->prepareBlacklistIpCheckUrl($ip);
+
+        $response = $this->get($url);
```

Comment:
even though php allows the dynamic change of types, assigning a new type to an existing variable is bad practice.
"$response"<=string vs. "$response"<= HetrixtoolsServiceResponse
I'll rename the two variables => $jsonResponse | $hetrixtoolsServiceResponse